packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -920,7 +920,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
       onScrollToItem((currentPendingValueValidIndex >= 0 || currentPendingValue !== '') ? currentPendingValueValidIndex : selectedIndex);
     } else {
       // We are using refs, scroll the ref into view
-      if (this._selectedElement && scrollSelectedToTop) {
+      if (this._selectedElement && this._selectedElement.offsetParent && scrollSelectedToTop) {
```

Comment:
I feel like these checks should be reworked so that we are only checking if selectedElement has an offsetParent once. It looks like every codepath requires that this offsetParent exists.